internal/report/excel-report.go

```diff
@@ -171,15 +247,32 @@ func fillSalesData(f *excelize.File, sheetName string, sales []repositories.Sale
 		rowCh <- struct {
 			row  int
 			sale repositories.SalesReport
-		}{row: index + 10, sale: sale}
+		}{row: index + 7, sale: sale}
 	}
 	close(rowCh)
 
 	wg.Wait()
-	return nil
+	return len(sales) + 7, nil
 }
 
 func setSalesRow(f *excelize.File, sheetName string, row int, sale repositories.SalesReport) error {
+	exp := "#,##0.00_);(#,##0.00)"
```

Comment:
los estilos no deberian crearce dentro de un for, ni aplicarse dentro de un for, deber aplicarse, luego de recorrer todos los datos, se deberia guardar la ultima fila para poder aplicar el formato una sola vez luego que ha terminado el llenado, si hay 1000 filas, y le damos un formato de numero a la columna A, seria desde A10-A1000 pero luego ya del proceso o antes.